examples/pxScene2d/src/pxResource.cpp

```diff
@@ -100,6 +101,8 @@ void pxResource::removeListener(pxResourceListener* pListener)
   {
     mInitialized = false;
     mDownloadRequest->setCallbackFunctionThreadSafe(NULL);
+
+    delete mDownloadRequest;
```

Comment:
Multiple objects with multiple threads have pointers to this pxFileDownloadRequest object (stored in mDownloadRequest).  It should be deleted in either pxFileDownloader::downloadFile()  or pxResource::onDownloadComplete() only to avoid double frees.